.travis.yml

```diff
@@ -6,6 +6,8 @@ os:
 julia:
   - 1.0
   - 1.1
+  - 1.2
```

Comment:
Do we want to keep testing on 1.1 and 1.2?

Reply:
yes, because one of the aim of JLSO is to be able to load data from old versions of Julia, and so it'd be nice to at least know it runs on the old version